ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -129,6 +129,17 @@ ORK_CLASS_AVAILABLE
  */
 + (BOOL)removePasscodeFromKeychain;
 
+/**
+ This method allows to force the passcode to be a certain value. You only want to use this capability when you're moving your existing users
+ from a custom passcode entry mechanism to the one provided by ResearchKit.
+ 
+ This method will raise an exception if `passcode` is nil.
+ 
+ @param passcode          The passcode to store
+ @param touchIdEnabled    Whether TouchId will be available during passcode entry
+ */
++ (void)forcePasscode:(NSString *)passcode withTouchIdEnabled:(BOOL)touchIdEnabled;
```

Comment:
I see. My only concern was that the method might be misunderstood and used incorrectly.
That being said, it is well-documented.
Unless anyone else has any other solutions, I am okay to proceed with this implementation.